### PR TITLE
[EI-12, EI-52, 6861] Edge Integrations Guide- WP

### LINKED ISSUE #6861: Edge Integrations
Re: Creating Edge Personalization Guide

Priority: High

## Issue Description
Create a guide for Edge Integrations 

## Additional Information
* Add reference to Edge Personalization to the following docs:

- [Pantheon Modules](https://pantheon.io/docs/modules)
- [Pantheon Plugins for WordPress](https://pantheon.io/docs/plugins)
- [Pantheon Global CDN](https://pantheon.io/docs/global-cdn) 
- [Advanced Global CDN | Pantheon Professional Services](https://pantheon.io/docs/guides/professional-services/advanced-global-cdn)
- [Caching: Advanced Topics](https://pantheon.io/docs/caching-advanced-topics)
- [Global CDN Caching for High Performance with Drupal and WordPress](https://pantheon.io/docs/global-cdn-caching)
- [Frontend Performance](https://pantheon.io/docs/guides/frontend-performance)

### PR DESCRIPTION
**Pull request by michellecolon-pantheon**
Closes #6861 

## Summary

**[Edge Integrations Guide](https://pantheon.io/docs/doc-title)** - WP-related additions to Intro and Config Overview sections.

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by michellecolon-pantheon** [comment]
Hi @jazzsequence, created a PR, if you want/need to add more. Otherwise, let me know and I'll merge!

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7166](https://pr-7166--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `57.99999999999999` | `0` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7166` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34616/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34616/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7166--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by jazzsequence** [comment]
@michellecolon-pantheon should be good to merge now. I don't think I can add you as a reviewer because you opened the PR. 😜

**Comment by michellecolon-pantheon** [comment]
@jazzsequence - Are we okay to display/make public the wp13n URL? If it is a potential vulnerability, I'm happy to mask it.

**Comment by jazzsequence** [comment]
@michellecolon-pantheon Good question. I'd say probably not a bad idea to mask it. I don't think it's sensitive information, per se, but I also don't know that we want to advertise it.

File: source/content/guides/edge-integrations/04-plugin-install.md [hunk 1]
```diff
@@ -14,46 +14,73 @@ editpath: edge-integrations/04-plugin-install.md
...
+
+### Install manually
+
+If you do not use Composer on your project at all, you can still get started with the WordPress Edge Integrations plugin without any hassle. In this case, you won't be installing the SDK package, instead, go to the [Pantheon WordPress Edge Integrations Releases page](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/releases).
+
+* Download the Source Code (zip) file associated with the most recent version.
+
+![Release Assets](../../../images/guides/edge-integrations/ei-wp-plugin-assets.png)
+
+* Extract the plugin in your `wp-content/plugins` directory. You will get all of the compiled assets and included dependencies, including the CMS-agnostic, [global PHP library](https://github.com/pantheon-systems/pantheon-edge-integrations) in the package.
```
[the remark this answers is not in the document]
Comment (jazzsequence) [reply source/content/guides/edge-integrations/04-plugin-install.md:46]: They would get everything except the documentation in the SDK, specifically.

[the remark this answers is not in the document]
Comment (michellecolon-pantheon) [reply source/content/guides/edge-integrations/04-plugin-install.md:46]: @jazzsequence Thank you!

File: source/content/guides/edge-integrations/04-plugin-install.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ editpath: edge-integrations/04-plugin-install.md
 reviewed: "2022-03-09"
 ---
 
-This doc will help you personalize, and provide custom experiences for visitors to your website, based on Geotargeting(geo) and Interest targeting.
+This doc will help you personalize, and provide custom experiences for visitors to your website, based on Geotargeting (geo) and Interest targeting.
```
Comment (scottbuscemi) [inline source/content/guides/edge-integrations/04-plugin-install.md:17]: Let's change this to just "Geotargeting" and remove the "(geo)" @jazzsequence

**Comment by scottbuscemi** [comment]
@jazzsequence On line 72, let's change the phrasing to state something like "This includes country, continent, region, city, and connection information (connection speed and connection type)"

**Comment by jspellman814** [comment]
There are duplicate lines: 41-53 and 55-61

File: source/content/guides/edge-integrations/04-plugin-install.md (hunk 2 above)
Comment (jazzsequence) [inline source/content/guides/edge-integrations/04-plugin-install.md:17]:
```suggestion
This doc will help you personalize, and provide custom experiences for visitors to your website, based on Geotargeting and Interest targeting.
```

**Comment by jazzsequence** [comment]
@michellecolon-pantheon Made some tweaks based on yesterday's call and pulled in the updated stuff from the original PR.

**Comment by jazzsequence** [comment]
@scottbuscemi @michellecolon-pantheon https://github.com/pantheon-systems/documentation/pull/7166/commits/5fd4780bccca0da8ea96dead9c4a60d0c2903aed adds the cookie consent management section

**Pull request merged**